docs/Configuration.md

```diff
@@ -0,0 +1,40 @@
+# Configuration
+
+* [Receivers](#receivers)
+* [Processors](#processors)
+* [Exporters](#exporters)
+
+## Receivers
+
+### Filelog Receiver
+
+Filelog Receiver simple config
+
+Link to [Filelog Receiver detailed configuration][filelogreceiver_readme]
+
+### Syslog Receiver
+
+Syslog Receiver simple config
+
+Link to [Syslog Receiver detailed configuration][syslogreceiver_readme]
+
+### Telegraf Receiver
+
+Telegraf Receiver simple config
+
+Link to [Telegraf Receiver detailed configuration][telegrafreceiver_readme]
+
+### OTLP Receiver
+
+OTLP Receiver simple config
+
+Link to [OTLP Receiver detailed configuration][otlpreceiver_readme]
+
+## Processors
+
+## Exporters
+
+[filelogreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.27.0/receiver/filelogreceiver
+[syslogreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.27.0/receiver/syslogreceiver
+[otlpreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector/tree/v0.27.0/receiver/otlpreceiver
```

Comment:
Why not provide link to our fork https://github.com/SumoLogic/opentelemetry-collector?

Reply:
because we are using the contrib ones

Reply:
Yeah, I basically looked into the builder definition and used what is in there.